json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -154,8 +154,7 @@
       "type": "string"
     },
     "startDate": {
-      "maxLength": 100,
-      "minLength": 1,
+      "format": "date",
```

Comment:
Again, if we don't change the DOS1 version in the frameworks repo then this doesn't need to change.